org.claws_mail.Claws-Mail.json

```diff
@@ -0,0 +1,161 @@
+{
+  "app-id": "org.claws_mail.Claws-Mail",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.32",
+  "sdk": "org.gnome.Sdk",
+  "command": "claws-mail",
+  "finish-args": [
+    "--socket=x11",
+    "--socket=wayland",
+    "--share=ipc",
+    "--share=network",
+    "--filesystem=home:rw",
```

Comment:
Is it for saving attachments?

Reply:
Claws-mail uses `~/.claws-mail` for configuration, plug-ins, metadata, in short any operational files. By default mail is stored in `~/Mail` maildir. A user can create maildirs and mbox-files at arbitrary locations. So, if I wouldn't bind `~` to the host, Claws-Mail would potentionally store data at locations that are transient. I would prefer to be very strict and expose individual directories as before, but I don't know how to secure the behavior to a fixed number of directories.

Reply:
Sounds good. It's not like we can do much about GTK2 apps anyway.